transition-guide.md

```diff
@@ -72,60 +74,63 @@ To transition and test your application, you will need a suitable Graviton envir
 
-During the transition to Graviton, if you are using Amazon EC2 Auto Scaling, you may be able to increase the threshold values for the CloudWatch alarms that invoke the scaling process. This may reduce the number of EC2 instances now needed to serve a given level of demand.
+During the transition to Ampere processor based instances, if you are using autoscaling provided by CSPs, you may be able to increase the threshold values for monitoring tool's alarms that invoke the scaling process. This may reduce the number of Ampere processor based instances now needed to serve a given level of demand.
 
 Important: This repository has sections dedicated to [Optimization](optimizing.md) and a [Performance Runbook](perfrunbook/graviton_perfrunbook.md) for you to follow during this stage.
 
-If after reading the documentation in this repository and following the recommendations you do not observe expected performance then please reach out to your AWS account team, or send email to [ec2-arm-dev-feedback@amazon.com](mailto:ec2-arm-dev-feedback@amazon.com) with details so we can assist you with your performance observations.
+If after reading the documentation in this repository and following the recommendations you do not observe expected performance then please reach out to your Ampere account team, or send email to [support@amperecomputing.com](mailto:support@amperecomputing.com) with details so we can assist you with your performance observations.
 
 
 ### Infrastructure and deployment
 
-**Step 7 - Update your infrastructure as code**
+**Step 7 - Apply Arm64 code to infrastructure**
 
-Now you have a tested and performant application, its time to update your infrastructure as code to add support for Graviton-based instances. This typically includes updating instance types, AMI IDs, ASG constructs to support multi-architecture (see [Amazon EC2 ASG support for multiple Launch Templates](https://aws.amazon.com/about-aws/whats-new/2020/11/amazon-ec2-auto-scaling-announces-support-for-multiple-launch-templates-for-auto-scaling-groups/)), and finally deploying or redeploying your infrastructure.
+Now you have a tested and performant application, its time to deploy Arm64 application to your infrastructure.
+If you are running applications on your own on-premise infrastructure, it's your choice to either apply the application by container orchestration like Kubernetes in a Ampere processor based cluster, or using Systemd to manage your application on a single Linux server.
```

Comment:
I am unsure where systemd comes into it - I would talk about updating Terraform, Ansible, Pulumi, etc code to use the correct instance types for creating Arm64 clusters or instances, and deploying or redeploying to a new architecture.

Reply:
I was thinking of a standalone condition. But you are right, users can manage bare-metal or local cluster by infrastructure as code tool as well. Updated per your suggestion, thanks.